### PR TITLE
Check that validator set is not empty at genesis

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

**Comment by bkchr** [comment]
Fixes: https://github.com/paritytech/substrate/issues/3064

File: srml/session/src/lib.rs [hunk 1]
```diff
@@ -316,6 +316,8 @@ decl_storage! {
 				let initial_validators = T::SelectInitialValidators::select_initial_validators()
 					.unwrap_or_else(|| config.keys.iter().map(|(ref v, _)| v.clone()).collect());
 
+				assert!(!initial_validators.is_empty(), "Empty validator set in genesis block!");
```
Comment (gavofyork) [inline srml/session/src/lib.rs]: No panickers allowed in the runtime :-)

**Comment by andresilva** [comment]
I don't think there's any need to bump `spec_version` or `impl_version`.

File: srml/session/src/lib.rs (hunk 1 above)
Comment (andresilva) [inline srml/session/src/lib.rs]: This panics but I think it's fine since it's in `add_extra_genesis` (i.e. it's not part of block execution in runtime)

**Pull request merged**